### ISSUE TITLE
Change `antidote` default path

### ISSUE THREAD

**Issue by IronicUsername** [instantOS/instantSHELL#7]
**Goal**
Have the path to antidote be flexible.

**Context**
When setting up a fresh instance of InstantOS i noticed that by default, the path to `antidote` was always searched under `$zshhome/.antibody`

Since `antidote` devs officially tell ppl you can [either install it by pulling git](https://github.com/mattmc3/antidote?tab=readme-ov-file#install-with-git) into `${ZDOTDIR:-$HOME}/.antidote` 
**OR**
[install via a package manager](https://github.com/mattmc3/antidote?tab=readme-ov-file#install-with-a-package-manager) which in turn installs `antidote` in a different path.

so basically here
https://github.com/instantOS/instantSHELL/blob/c1d559ad5fd444a454e5b6e5d4739149e5f435a4/zshrc#L34-L57

**Tasks**
- [ ] the `.zshrc` should:
  - [ ]  check if `antidote` is ... 
    - [ ] not installed,
      - [ ] install on system via git (maybe also following XDG Base Directory specs if possible...)
      - [ ] use it for the rest of the function
  - [ ] installed,
    - [ ] "find" the path of the installed version & use it
